pkg/renderer/context.go

```diff
@@ -155,6 +164,15 @@ func (ctx *Context) GetImagesDir() string {
 	return ""
 }
 
+// MacroTemplate finds and returns a user macro function by specified name.
+func (ctx *Context) MacroTemplate(name string) (MacroTemplate, error) {
+	macro, ok := ctx.macros[name]
+	if ok {
+		return macro, nil
+	}
+	return nil, errors.New("Unknown user macro: " + name)
```

Comment:
please avoid capital letters at the beginning of the error message. If the messages are wrapped/concatenated, we will have capital letters in the middle of the sentence.